main.go

```diff
@@ -86,18 +82,33 @@ func main() {
 	s.ListenAndServe()
 }
 
+func newProxy(backend *url.URL, l log.Logger) *httputil.ReverseProxy {
```

Comment:
I personally prefer having the loggers, db and such as first arguments. Not sure what the convention is (if there is any).